nibabel/nifti1.py

```diff
@@ -232,6 +233,17 @@
     (2003, 'rgb vector', (), "NIFTI_INTENT_RGB_VECTOR"),
     (2004, 'rgba vector', (), "NIFTI_INTENT_RGBA_VECTOR"),
     (2005, 'shape', (), "NIFTI_INTENT_SHAPE"),
+    # FSL-specific intent codes - FNIRT
```

Comment:
Oh, sure - the FNIRT codes can be found in `$FSLDIR/src/warpfns/fnirt_file_reader.h:104`, and the TOPUP codes in `$FSLDIR/src/topup/topup_file_io.h:104`. Note that this is for 5.0.9 - I haven't downloaded 5.0.10 yet (released a few days ago), so they may have changed (although probably haven't). I will check tomorrow, and update here if the line numbers have changed in 5.0.10.